spec/system/cytoskeletal_form_submission_spec.rb

```diff
@@ -71,13 +74,20 @@
       fill_in "publisher", with: publisher
       fill_in "publication_year", with: 2019
       find("#collection_id").find(:xpath, "option[1]").select_option
+      click_on "v-pills-additional-tab"
+      fill_in "related_identifier_1", with: related_identifier
+      fill_in "related_identifier_type_1", with: related_identifier_type
+      fill_in "relation_type_1", with: relation_type
       click_on "v-pills-curator-controlled-tab"
       fill_in "doi", with: doi
       fill_in "ark", with: ark
       click_on "Create"
       expect(page).to have_content "marked as draft"
       cytoskeletal_work = Work.last
       expect(cytoskeletal_work.title).to eq title
+      expect(cytoskeletal_work.resource.related_objects.first.related_identifier).to eq related_identifier
+      expect(cytoskeletal_work.resource.related_objects.first.related_identifier_type).to eq related_identifier_type
+      expect(cytoskeletal_work.resource.related_objects.first.relation_type).to eq relation_type
```

Comment:
Might assign `cytoskeletal_work.resource.related_objects.first` to a variable, mostly just to keep the line length down.